src/CLMWrapper.cpp

```diff
@@ -576,11 +570,30 @@ void ClmWrapper::callback(const sensor_msgs::ImageConstPtr& msgIn)
     active_models_st += active_m_C;
     cv::putText(disp_image, active_models_st, cv::Point(10,60), CV_FONT_HERSHEY_SIMPLEX, 0.5, CV_RGB(255,0,0));   
 
-    if(!clm_parameters[0].quiet_mode)
+    if (faceDetected)
     {
-        namedWindow("tracking_result",1);   
-        cv::imshow("tracking_result", disp_image);
+        // feeding disp_image into the cv pointer's image
+        // only if a face is detected
+        cv::cvtColor(disp_image,cv_ptr->image, CV_RGB2BGR);
     }
+     //publishing the image usign the cv pointer
+    imagePublisher.publish(cv_ptr->toImageMsg());
```

Comment:
disp_image and capture_image are of type 'cv::Mat', so I think I can't publish them through imagePublisher because it's supposed to transfer Image files by definition, line 673:
`imagePublisher = imageTransport.advertise`. That's why I used cv_ptr + toImageMsg(). Now, I will use CvBridge + captured_image instead